inc/model.class.php

```diff
@@ -1497,10 +1504,59 @@ static function showLogResults($models_id) {
                echo "<td>".nl2br($result['status_message'])."</td>";
                echo "<td>".$result['type']."</td>";
                echo "<td>".$result['url']."</td><tr>\n";
+               if(!isset($resume[$result['status']][$result['type']])) {
+                  $resume[$result['status']][$result['type']] = 0;
+               }
+               $resume[$result['status']][$result['type']]++;
+               $nblines++;
             }
             echo "</table></div>\n";
             echo "<script type='text/javascript'>document.getElementById('log1_table').style.display='none'</script>";
          }
+
+         if (!empty($resume)) {
+            echo "<table>\n";
+            echo "<tr>";
+            echo "<td style='width:30px'>";
+            echo "<a href=\"javascript:show_log('2')\"><img src='../pics/minus.png' alt='minus' id='log2'>";
```

Comment:
`id="log2"` is already used.
I guess you should use `id="log3"` and `<div id='log3_table'>`